src/app/projects/pages/projects-list/projects-list.component.html

```diff
@@ -154,7 +154,7 @@
               </ul>
             </li>
             <li *ngIf="project.geoAccessions.length">
-              GEO:
+              GEO: <br />
```

Comment:
Why the line break? Generally, you should use CSS to style this kind of thing

Reply:
How do you do that with CSS?
Line break because all the other accession were line breaking apart from the new ones, because of coincidental spacing. So to make it consistent, add the line breaks to stop it looking weird.